### DISCUSSION TITLE
[A-L] (2023/24) Foglio 14 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#283] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/08cc637c-4c09-418d-9ca5-a0393efc85a7)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/6225f507-47df-49ef-bd2a-4b4b54b2dd51)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/2f68bb52-a6e9-4780-afc8-a9b9c2dbdaec)

**Comment by daromacarlo** [comment]
![Immagine 2023-12-07 121341](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/7da0f17c-3ea5-4e8a-b6fe-88a4cb9e14ed)
![Immagine 2023-12-07 121357](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/178d68b5-d042-4478-b04d-6b0dd4bf3673)

**Comment by CiottoloMaggico** [comment]
Soluzioni dell'omogenea prese da esercizio 1 #116 
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/f5525542-5a64-4110-bca0-960187958900)
